### ISSUE TITLE
Move Products List above Payment Details on Purchase Confirmation Page

### ISSUE THREAD

**Issue by psahalot** [easydigitaldownloads/easy-digital-downloads#5805]
I have been a long time EDD user for my WP themes and plugins. 
The most common question I have received in support requests is "How do I download the plugin/theme? 
There are two reasons for this: 

1. Sometimes email receipts do not reach the mailbox due to hosting server's problems. I have had this problem even with DO servers. 
2. The purchase confirmation page lists the downloads below the payment details and these links go unnoticed. 

In my opinion, this design can be improved. 

If we look at the Purchase Confirmation of EDD, we see that we have all the promotional stuff visible in the first half. Then we have Payment and Subscription details. Finally, at the bottom, we have the Downloads. 
![screen shot 2017-06-13 at 10 57 33 am](https://user-images.githubusercontent.com/1639806/27067487-bd26d9ea-5028-11e7-8d6b-556910bbc095.png)

### Suggested Improvement:

**I think, as a buyer, I would like to get access to my products in the first place.** 
![screen shot 2017-06-13 at 11 10 12 am](https://user-images.githubusercontent.com/1639806/27067526-e728c000-5028-11e7-950e-0fbf9df135eb.png)

**Comment by cklosowski** [comment]
@psahalot So ignoring our specific purchase confirmation on the EDD site (since we add the promotional stuff at the top, that does not ship with EDD Core like that). This is the plain view:
![screen shot 2017-06-13 at 9 44 49 am](https://user-images.githubusercontent.com/1390124/27093861-0cffd748-501d-11e7-96c3-aa15a029e633.png)

It would just be swapping the order.

@easydigitaldownloads/core-devs thoughts on that?

**Comment by psahalot** [comment]
@cklosowski Yes, I am aware of the plain view. I used EDD page as a reference to mention that the actual products get almost hidden when we add some custom content to the page. We have a similar setup on our sites as well where we ask users to connect with us on Twitter and Facebook. 

Moving the Products to the top is all we need :)

**Comment by brashrebel** [comment]
@psahalot is your need satisfied by just [overriding the template in your theme](http://docs.easydigitaldownloads.com/article/1216-moving-edd-templates-to-your-theme)? This would be `shortcode-receipt.php`. Or are you advocating for this becoming a permanent EDD core change?

**Comment by psahalot** [comment]
@brashrebel I agree that I can do this with template override, which I have done as well. But, I think, it should be a part of EDD core for better UX. I can submit a PR, if you would like me to do so :)

**Comment by cklosowski** [comment]
I think we can work something up for it, I just wanted it recorded here that we need to look at the simplest example as to not confuse people stumbling upon the issue.

**Comment by SDavisMedia** [comment]
I agree with this change.

**Comment by psahalot** [comment]
Thanks @SDavisMedia  :)
